File: Backend/TopSys.TopConWeb.Domain/Entities/Endereco.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TopSys.TopConWeb.Domain.Entities
{
    public class Endereco
    {
        public string Cep { get; set; }

        public string Logradouro { get; set; }

        public int Numero { get; set; }

        public string Complemento { get; set; }

        public string Bairro { get; set; }

        public int MunicipioCodigo { get; set; }

        public virtual Municipio Municipio { get; set; }

        public bool IsConfiavel { get; set; }
    }
}

File: Frontend/src/app/classes/endereco/endereco.ts
export class Endereco {
    cep: string = '';
    logradouro: string = '';
    numero: number = 0;
    complemento: string = '';
    bairro: string = '';
    municipio: Municipio; // = new Municipio()
}

export class Municipio {
    codigo: number = 0;
    nome: string = '';
    uf: string = '';
    ibgeCodigo: number = 0;
}

export class DistanciaViaGoogleApi {
    distanciaEmKm: number = 0;
    utilizaGoogleApi: boolean = true;
}